### PR TITLE
Add double chromosome X and single chromosome Y

### PR DESCRIPTION
**Pull request by cytham**
I am very thankful for this utility as it allows easy visualization of chromosomes. I am currently integrating it as a graphical output for my genomic copy-number calling tool.

Here are some changes I made for an improvement of tagore, particularly to include chrY and a second chrX.

To include the extra chromosomes, I have modified the COORDINATES variable in tagore main executable file, and have also modified the base.svg.p template for the svg creation. Below is an image of the new output.

I hope that these changes will be accepted so tagore can be integrated into my tool and be more useful to the wider community. 

If not, I will like to seek your permission to include the modified tagore script and base.svg.p within my tool which I will be publishing soon. I will state the use of tagore upfront and fully acknowledge both @ar0ch and @lavanyarishishwar for their work on this tool.

Hope to hear from you soon.

Thanks!

out

### CODE REVIEW

File: tagore [hunk 1]
```diff
@@ -16,7 +16,7 @@ import sys
 from argparse import ArgumentParser, HelpFormatter
 
 
-VERSION = '1.0.2'
+VERSION = '1.0.2a'
```
Comment (ar0ch) [inline tagore:19]: Please bump the version to 1.1.0

**Review by ar0ch — CHANGES_REQUESTED** [review]
Thanks for this PR! Could you bump the version to 1.1.0?

File: tagore (hunk 1 above)
Comment (cytham) [reply tagore]: thanks for the fast reply. Sure, just did!

**Review by ar0ch — APPROVED** [review]

File: tagore (hunk 1 above)
Comment (cytham) [reply tagore]: Thanks! Yes, that's a good point. An error out might be problematic in cases where a genome really has XXY or other unusual combinations. A warning will be good.

**Comment by cytham** [comment]
@ar0ch thanks for approving the PR!
I am wondering if you can bump the 1.1.0 version to PyPI and Conda also? 
Thanks!

**Comment by ar0ch** [comment]
Just uploaded it to PyPI, and its in the queue at bioconda

**Pull request merged**